### DISCUSSION TITLE
Can't access head elements like `meta`

### DISCUSSION THREAD

**Discussion by raphaklaus** [keathley/wallaby#660] [Q&A]
Hello! I am trying to select some head elements like `meta` but receive this message: `     ** (Wallaby.QueryError) Expected to find 1, visible element that matched the css 'meta' but 0, visible elements were found.`

Code that I used:

```
head = session
    |> visit("https://google.com")
    |> find(Query.css("meta"))
```

If I ask the `session` to do a `take_snapshot` or `page_source` I can see the page rendered and the meta tag in the source, but can't achieve the querying when it's inside `head` elements. Selecting body elements works nicely though.

Using version `0.29.1`

**Comment by mhanberg** [comment]
Hi, 

By default, selectors only return visible elements. This ensures that your test is operating the page as a human user would. If you need to find elements that are not visible, you can add `visible: false` to your query.

```elixir
head = session
    |> visit("https://google.com")
    |> find(Query.css("meta", visible: false, count: :all))
```

In the above example, I also used the option `count: :all` so that the finder would return all of the `meta` elements, or else the test would most likely find more than one and raise, as by default queries are scoped to find a single element.
